siac/hostcmd.go

```diff
@@ -130,9 +130,9 @@ func hostcmd() {
 	}
 
 	// Determine the competitive price string.
-	ah := new(api.ActiveHosts)
+	ah := new(api.HostdbActiveGET)
 	var competitivePrice string
-	err = getAPI("/hostdb/active?numhosts=24", ah)
+	err = getAPI("/hostdb/active?numhosts=32", ah)
```

Comment:
nvm I didn't realize this was for the host competitive price estimation, and not for the renter side of things. Host competitive price estimator intentionally undershoots a bit to account for potential other factors that may affect host score.